Oswaldo Vazquez Aparicio/30 Agosto/hackerrank/JavaArrayList.java

```diff
@@ -6,7 +6,7 @@
 public JavaArrayList {
 
     private static final Scanner scanner = new Scanner(System.in);
-    private static ArrayList<ArrayList<Integer>> table;
+    private static List<List<Integer>> table;
```

Comment:
When you variable is a Collection, it is good practice put name in singular and them the the type, example: tableListList, userSet, etc...